model/src/authenticated/account/raids.rs

```diff
@@ -0,0 +1,12 @@
+use crate::{Endpoint, FixedEndpoint};
+
+pub type RaidEvent = Vec<String>;
```

Comment:
`RaidEvent` should be of type `Vec<(Raid)EventId>` and you should define `Event::id` to be the same type